docs/amber_script.md

```diff
@@ -229,26 +231,56 @@ value for `START_IDX` is 0. The default value for `COUNT` is the item count of
 vertex buffer minus the `START_IDX`.
 
 ```
+# Run the given |pipeline_name| which must be a `compute` pipeline. The
+# pipeline will be run with the given workgroup |x|, |y|, |z| dimensions.
```

Comment:
Actually it's the number of workgroups in the x, y, and z dimensions.  See vkCmdDispatch.

The workgroup size is specified in the SPIR-V module by a LocalSize execution mode (or a variable that has specializable components.)